lib/babel.js

```diff
@@ -24,11 +23,9 @@ var powerAssert = createEspowerPlugin(babel, {
 	patterns: require('./enhance-assert').PATTERNS
 });
 
-// if generators are not supported, use regenerator
 var options = {
-	blacklist: hasGenerator ? ['regenerator'] : [],
-	optional: hasGenerator ? ['asyncToGenerator', 'runtime'] : ['runtime'],
```

Comment:
It was included in that preset, that's why I dropped it. Every intention of adding it back.

We are still waiting on https://phabricator.babeljs.io/T6644. I will approach this again when that closes.